docs/getting-started-guides/docker-multinode.md

```diff
@@ -0,0 +1,261 @@
+### Running Multi-Node Kubernetes Using Docker
+
+_Note_: These instructions are somewhat significantly more advanced than the [single node](docker.md) instructions.  If you are
+interested in just starting to explore Kubernetes, we recommend that you start there.
+
+
+## Master Node
+We'll begin by setting up the master node.  For the purposes of illustration, we'll assume that the IP of this machine is MASTER_IP
+
+### Setup Docker-Bootstrap
+We're going to use ```flannel``` to set up networking between Docker daemons.  Flannel itself (and etcd on which it relies) will run inside of
+Docker containers themselves.  To achieve this, we need a separate "bootstrap" instance of the Docker daemon.  This daemon will be started with
+```--iptables=false``` so that it can only run containers with ```--net=host```.  That's sufficient to bootstrap our system.
+
+Run:
+```sh
+sudo sh -c 'docker -d -H unix:///var/run/docker-bootstrap.sock -p /var/run/docker-bootstrap.pid --iptables=false --ip-masq=false --bridge=none --graph=/var/lib/docker-bootstrap 2> /var/log/docker-bootstrap.log 1> /dev/null &'
+```
```

Comment:
We should suggest people run the bootstrap docker instance under something like upstart or systemd to ensure it comes back after reboot.